docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -174,12 +174,14 @@ These elements allow the user to define the DomainParticipant configuration.
     :language: xml
     :start-after: <!-->XML-PARTICIPANT<-->
     :end-before: <!--><-->
-    :lines: 2-3, 5-76, 78
+    :lines: 2-3, 5-86, 88
```

Comment:
Again, the end of the XML file is not the correct one (it is not correct in the current documentation version either). I suggest fixing it in this PR

Reply:
2ab1b34